docs/guides/Localization and Internationalization.md

```diff
@@ -47,7 +48,7 @@ The upcoming road or ramp destination is named according to the local or nationa
 | Hebrew     | ✅              | ✅                             | —                     | ✅
 | Hungarian  | ✅              | —                             | —                     | ✅
 | Italian    | ✅              | ✅                             | ✅                     | ✅
-| Portuguese | ✅              | ✅                             | ✅                     | ✅
+| Portuguese | ✅             | ✅                             | ✅                     | ✅
```

Comment:
Huh, so we noted we already had Portuguese before this addition?

Reply:
Yes, we already have a Brazilian Portuguese localization. I considered spelling out the availability of both variants, but that might make more sense once we make European Portuguese spoken instructions available too.